README.md

```diff
@@ -35,10 +35,36 @@ following files:
   1. as final step, return the bare-metal machine to the CentOS CI for other
      users (it will be re-installed with a minimal CentOS environment again)
 
+- `e2e.groovy` is the Jenkins Pipeline responsible for running e2e tests on
+  a multi-node kubernetes cluster hosted on [Centos CI](https://wiki.centos.org/QaWiki/CI).
+  It verifies complete e2e functionalities for a corresponding Pull request
+  on [ceph-csi](https://github.com/ceph/ceph-csi/tree/master) repository.
+  It executes the following stages:
+
+  1. dynamically allocate a Jenkins Slave (`node('cico-workspace')`) with tools
+     and configuration to request a bare-metal machine.
+  2. checkout the `centos/ci` branch of the repository, which contains scripts
```

Comment:
you can just number everything with `1.` and markdown will number them correctly